src/com/goide/psi/impl/GoElementFactory.java

```diff
@@ -214,4 +216,38 @@ private static String prepareVarOrConstDeclarationText(@NotNull String name, @Nu
     value = StringUtil.isNotEmpty(value) ? " = " + value : "";
     return name + type + value;
   }
+
+  public static GoStructType createMultipleFieldsFromSingleDeclaration(@NotNull Project project,
```

Comment:
Another thing that I'm not sure of is this. Should this be done for all the structure, regardless of the FieldDeclaration on which the quickfix was invoked on?